cmd/tigerblood/main.go

```diff
@@ -4,11 +4,23 @@ import (
 	"fmt"
 	"github.com/DataDog/datadog-go/statsd"
 	"go.mozilla.org/tigerblood"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
 )
 
+func loadCredentials() (map[string]string, error) {
+	creds := make(map[string]string)
```

Comment:
Won't we need more configuration in the future? It might be worth making this a generic conf loader rather than a specific credentials loader.